### ISSUE TITLE
unmet peer eslint@"^6.2.0 || ^7.0.0 || ^8.0.0": found 9.0.0

### ISSUE THREAD

**Issue by ihwf** [vuejs/eslint-plugin-vue#2443]
**Checklist**

- [x] I have tried restarting my IDE and the issue persists.
- [x] I have read the [FAQ](https://eslint.vuejs.org/user-guide/#faq) and my problem is not listed.

**Tell us about your environment**

- **ESLint version:** "^9.0.0"
- **eslint-plugin-vue version:** "^9.24.0"
- **Vue version:** "^3.4.21"
- **Node version:** v20.11.1
- **Operating System:** MacOS 12.7.3

**Please show your full configuration:**

```json5
import pluginVue from 'eslint-plugin-vue'
export default [
  // add more generic rulesets here, such as:
  // js.configs.recommended,
  ...pluginVue.configs['flat/recommended'],
  // ...pluginVue.configs['flat/vue2-recommended'], // Use this if you are using Vue.js 2.x.
  {
    rules: {
      // override/add rules settings here, such as:
      // 'vue/no-unused-vars': 'error'
    }
  }
]
```

**What did you do?**	

```vue
pnpm i
```

**What did you expect to happen?**
no WARN

**What actually happened?**

```
WARN  Issues with peer dependencies found
.
└─┬ eslint-plugin-vue 9.24.0
  └── ✕ unmet peer eslint@"^6.2.0 || ^7.0.0 || ^8.0.0": found 9.0.0
```

**Repository to reproduce this issue**

maybe just change dependencies config will resolve it

**Comment by xsjcTony** [comment]
+1, any plan for ESLint v9 support?

**Comment by FloEdelmann** [comment]
Since support for the new configuration format was already added in the [previous release](https://github.com/vuejs/eslint-plugin-vue/releases/tag/v9.24.0), there is actually not much to do to support ESLint v9; it's only the peer dependency range that has to be updated. I created a pull request: #2444

**Comment by FloEdelmann** [comment]
@ihwf @xsjcTony this has been fixed in https://github.com/vuejs/eslint-plugin-vue/releases/tag/v9.24.1.
